TypeCobol/Compiler/SqlCodeElements/SqlStatementElement.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using TypeCobol.Compiler.CodeElements;
+
+namespace TypeCobol.Compiler.SqlCodeElements
+{
+    /// <summary>
+    /// Base class for all Sql statement CodeElements
+    /// </summary>
+    public abstract class SqlStatementElement : StatementElement
+    {
+        protected SqlStatementElement(CodeElementType codeElementType, StatementType statementType) 
+            : base(codeElementType, statementType)
+        {
+        }
+
+        /// <summary>
+        /// Type of executable Sql statement
+        /// </summary>
+        public StatementType SqlStatementType => base.StatementType;
```

Comment:
The property does nothing, please remove it.

Reply:
done in [7f2afb7](https://github.com/TypeCobolTeam/TypeCobol/pull/1928/commits/7f2afb778747d8f26ff0ce751781c2ab8845aa78)